packages/bpk-component-tooltip/stories.js

```diff
@@ -91,7 +93,9 @@ storiesOf('bpk-component-tooltip', module)
         id="my-tooltip"
         target={<a href="//skyscanner.net" target="_blank" rel="noopener noreferrer">Click me!</a>}
       >
-        Find cheap flights here! ✈️
+        Find cheap flights here!
+        <span role="img" aria-label="Plane">✈️</span>
```

Comment:
Was this change caught by the linter?

Reply:
It was! See here: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/accessible-emoji.md

Reply:
That's great :)